pom.xml

```diff
@@ -106,6 +106,9 @@
     <module>external/kafka-0-10-assembly</module>
     <module>external/kafka-0-10-sql</module>
     <module>external/avro</module>
+    <module>graph/api</module>
+    <module>graph/cypher</module>
+    <module>graph/graph</module>
```

Comment:
Do we need to add these modules to https://github.com/apache/spark/blob/master/dev/sparktestsupport/modules.py?

Reply:
Thanks for the hint. We wanted to add the modules for Python testing together with https://issues.apache.org/jira/browse/SPARK-27306, https://issues.apache.org/jira/browse/SPARK-27307 and https://issues.apache.org/jira/browse/SPARK-27308